memsource/api_rest/auth.py

```diff
@@ -0,0 +1,14 @@
+from memsource import models, api_rest
+
+
+class Auth(api_rest.BaseApi):
+    # Document: https://cloud.memsource.com/web/docs/api#tag/Authentication
+
+    def login(self, user_name: str, password: str) -> models.Authentication:
+        response = self._post("v1/auth/login", {
+            "userName": user_name,
+            "password": password,
+        })
+        response["user"] = models.User(response.pop("user"))
```

Comment:
```suggestion
        response["user"] = models.User(response["user"])
```
I think we don't need to pop `user`. Replacing it should work.